plugins/pagerduty/komand_pagerduty/actions/create_user/action.py

```diff
@@ -16,20 +16,36 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        """Run action"""
-        self.logger.info("Creating user %s", params)
-        user = pypd.User.create(data=params, from_email=params["from_email"])
-
-        if user:
-            user = normalize_user(user.json)
-        else:
-            user = None
-
-        self.logger.debug("Returned: %s", user)
-
-        return {"success": not not user, "user": user or empty_user}
-
-    def test(self):
-        """Test action"""
+        """Trigger event"""
+
+        # required
+        from_email = params.get("from_email")
```

Comment:
```suggestion
        from_email = params.get(INPUT.FROM_EMAIL)
```
and so on for every other input